virtual-server/examples/nodejs/virtual-server-client-examples/main.js

```diff
@@ -0,0 +1,104 @@
+const VSClient = require("./client.js")
+
+// A sample VirtualServer manifest
+const virtualServerManifest = `
```

Comment:
This should be a JSON object, submitting a YAML string is not very programattic.

Reply:
done: https://github.com/coreweave/kubernetes-cloud/pull/7/commits/d26b26251ed1d193896dd649d4fa7e35562c4da0